src/appleseed/renderer/kernel/shading/shadingresult.h

```diff
@@ -72,6 +72,8 @@ class ShadingResult
 
     // Apply alpha premultiplication to the main output and the AOVs.
     void apply_alpha_premult();
+
+    void set_to_opaque_pink();
```

Comment:
`set_main_to_opaque_pink()` since it does not touch AOVs (in particular it doesn't reset `m_aov_count`).